src/DynamoCoreWpf/Utilities/CrashUtilities.cs

```diff
@@ -26,28 +29,48 @@ internal static string GithubNewIssueUrlFromCrashContent(object crashContent)
             return baseUri.ToString();
         }
 
+        /// <summary>
+        /// Converts packages information into markdown formatfor use in Github issue body
+        /// </summary>
+        /// <param name="packagesNames">List of names of loaded packages</param>
+        /// <returns>A markdown format string to use in issue body.</returns>
+        internal static string PackagesNamesToMakrdown(IEnumerable<string> packagesNames)
+        {
+            string markdownText = "";
+            if (packagesNames != null)
+            {
+                markdownText = "## Loaded Packages" + Environment.NewLine;
+                foreach (var name in packagesNames)
+                {
+                    markdownText += "- " + name + Environment.NewLine;
+                }
+            }
+            return markdownText;
+        }
+
         /// <summary>
         /// Formats crash details and adds metadata for use in Github issue body
         /// </summary>
         /// <param name="details">Crash details, such as a stack trace.</param>
         /// <returns>A formatted, but not escaped, string to use as issue body.</returns>
         private static string GitHubCrashReportBody(object details)
         {
-            var stackTrace = details?.ToString() ?? string.Empty;
+            var markdownPackages = details?.ToString() ?? string.Empty;
 
             // This functionality was not available prior to version 2.1.0, so it should be the fallback value
             var dynamoVersion = AssemblyHelper.GetDynamoVersion().ToString() ?? "2.1.0+";
 
-            return BuildMarkdownContent(dynamoVersion, stackTrace);
+            return BuildMarkdownContent(dynamoVersion, markdownPackages);
         }
 
         /// <summary>
         /// Builds a Markdown string that will be posted to our new GitHub issue
         /// </summary>
         /// <param name="dynamoVersion">Dynamo version that should be recorded in the issue report</param>
         /// <param name="stackTrace">The crash stack trace to be included in the issue report</param>
```

Comment:
Let's remove this line of the param is no longer passed

Reply:
oops missed that, done!